templates/common/builtinvar/builtin_vars.go

```diff
@@ -68,3 +68,15 @@ func NamesInScope(f features.Features) []string {
 
 	return out
 }
+
+// ValidateBuiltinVars returns error if any of the attemptedNames are not valid builtin
+// var names.
+func ValidateBuiltinVars(attemptedNames []string) error {
```

Comment:
The set of builtin vars depends on the api_version, so we can't hardcode it like this. But the good news is that the function that we need already exists, it's the `Validate()` function on line 49 of this file.

Reply:
done